### PR TITLE
add registration and login pages

### PR DESCRIPTION
**Pull request by AnatoliAgeichik**
- add the view for registration and login pages
 - add router
 - add a store with vuex

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<script>
+  export default {
+    computed : {
+      isLoggedIn : function(){ return this.$store.getters.isLoggedIn}
+    },
+    methods: {
+      logout: function () {
+        this.$store.dispatch('auth/logout')
+        .then(() => {
+          this.$router.push('/login')
```
Comment (artsiom-streetshares) [inline src/App.vue]: This means after I logout you redirect me to login? If yes, I guess it's not best UX, if say I want to browse public doctors list unauthenticated.

File: src/services/auth.service.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:8080/api/auth/';
```
Comment (artsiom-streetshares) [inline src/services/auth.service.js]: Can the url be taken from environment during build?

File: src/services/auth.service.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      });
+  }
+
+  logout() {
+    localStorage.removeItem('user');
+  }
+
+  register(user) {
+    console.log(user);
+    return axios.post(API_URL + 'signup', {
```
Comment (artsiom-streetshares) [inline src/services/auth.service.js]: Should we save here user to local storage?

File: src/store/auth.module.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
+
+const user = JSON.parse(localStorage.getItem('user'));
+const initialState = user
+  ? { status: { loggedIn: true }, user }
```
Comment (artsiom-streetshares) [inline src/store/auth.module.js]: Not sure but should it be user: user?

File: src/store/auth.module.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  : { status: { loggedIn: false }, user: null };
+
+export const auth = {
+  namespaced: true,
+  state: initialState,
+  actions: {
+    login({ commit }, user) {
+      return new Promise((resolve, reject) => {
+        commit('auth_request');
+        axios({ url: 'http://localhost:5000/login', data: user, method: 'POST' })
```
Comment (artsiom-streetshares) [inline src/store/auth.module.js]: Use centralized api url variabe

**Review by artsiom-streetshares — APPROVED** [review]

**Review by MariaAfanasyeva — APPROVED** [review]

**Pull request merged**